test/libsolidity/SemanticTest.h

```diff
@@ -14,13 +14,13 @@
 
 #pragma once
 
-#include <test/libsolidity/util/TestFileParser.h>
```

Comment:
The original order was correct: From specific to generic.

Reply:
Yup, should revert back to what it was